### ISSUE TITLE
can't install reviewdog command on Travis-ci

### ISSUE THREAD

**Issue by fortkle** [haya14busa/reviewdog#43]
Errors when I try to install reviewdog command on travis-ci:
```
$ go get github.com/haya14busa/reviewdog/cmd/reviewdog
package github.com/haya14busa/reviewdog/cmd/reviewdog
	imports context: unrecognized import path "context"

The command "go get github.com/haya14busa/reviewdog/cmd/reviewdog" failed and exited with 1 during .
```

maybe, it's cause that  Golang version on travis-ci is `1.4` .  (I'm using travis-ci for PHP project. so, i can't change golang version easily.)
ref. https://github.com/go-swagger/go-swagger/issues/679

**Comment by haya14busa** [comment]
Thank you for the report and sorry for the inconvenience. I fixed it by #44 

But, I also recommend to use [the binary release](https://github.com/haya14busa/reviewdog/releases) like this
https://github.com/haya14busa/reviewdog/blob/master/circle.yml#L7

**Comment by fortkle** [comment]
Thanks for the quick reply and fix.
It's completely work!
